proprietary-files.txt

```diff
@@ -435,85 +703,29 @@ vendor/lib/libmmiipstreammmihttp.so
 vendor/lib/libmmipstreamnetwork.so
 vendor/lib/libmmipstreamsourcehttp.so
 vendor/lib/libmmipstreamutils.so
-vendor/lib/libmmosal.so
-vendor/lib/libmmparser.so
-vendor/lib/libmmparser_lite.so
-vendor/lib/libmmrtpdecoder.so
-vendor/lib/libmmrtpencoder.so
 vendor/lib/libmmsw_detail_enhancement.so
 vendor/lib/libmmsw_math.so
 vendor/lib/libmmsw_opencl.so
 vendor/lib/libmmsw_platform.so
 vendor/lib/libmodalityservice_jni.so
-vendor/lib/libnetmgr.so
-vendor/lib/liboemcrypto.so
 vendor/lib/libomx-dts.so
 vendor/lib/libpanelmode.so
-vendor/lib/libperipheral_client.so
 vendor/lib/libprotobuf-cpp-biss.so
 vendor/lib/libprotobuf-cpp-shit.so
-vendor/lib/libq3dtools_adreno.so
-vendor/lib/libqcci_legacy.so
-vendor/lib/libqcmaputils.so
-vendor/lib/libqdi.so
-vendor/lib/libqdp.so
-vendor/lib/libqisl.so
-vendor/lib/libqmi.so
-vendor/lib/libqmi_cci.so
-vendor/lib/libqmi_client_helper.so
-vendor/lib/libqmi_client_qmux.so
-vendor/lib/libqmi_common_so.so
-vendor/lib/libqmi_csi.so
-vendor/lib/libqmi_csvt_srvc.so
-vendor/lib/libqmi_encdec.so
-vendor/lib/libqmiservices.so
 vendor/lib/libqti-at.so
 vendor/lib/libqti-gt.so
-vendor/lib/libqti-iop-client.so
-vendor/lib/libqti-iop.so
-vendor/lib/libqti-perfd-client.so
-vendor/lib/libqti_performance.so
-vendor/lib/librcc.so
 vendor/lib/libremosaic_daemon.so
-vendor/lib/librpmb.so
-vendor/lib/librs_adreno.so
-vendor/lib/librs_adreno_sha1.so
 vendor/lib/libscalar.so
 vendor/lib/libscale.so
 vendor/lib/libsd_sdk_display.so
-vendor/lib/libsdm-color.so
-vendor/lib/libsdm-diag.so
-vendor/lib/libsdm-disp-vndapis.so
-vendor/lib/libsdmextension.so
-vendor/lib/libsdsprpc.so
-vendor/lib/libsecureui_svcsock.so
 vendor/lib/libseemore.so
-vendor/lib/libsettings.so
 vendor/lib/libslimclient.so
 vendor/lib/libsmemlog.so
-vendor/lib/libssd.so
 vendor/lib/libsubsystem_control.so
-vendor/lib/libsystem_health_mon.so
-vendor/lib/libthermalclient.so
-vendor/lib/libthermalioctl.so
-vendor/lib/libtinyxml2_1.so
 vendor/lib/libuy.so
 vendor/lib/libvendorconn.so
 vendor/lib/libvoice-svc.so
-vendor/lib/libvpplibrary.so
-vendor/lib/libvpptestutils.so
-vendor/lib/libvqzip.so
 vendor/lib/libwbc_jni.so
-vendor/lib/libwfdavenhancements.so
-vendor/lib/libwfdcommonutils.so
-vendor/lib/libwfdhdcpcp.so
-vendor/lib/libwfdmmsink.so
-vendor/lib/libwfduibcsink.so
-vendor/lib/libwfduibcsinkinterface.so
-vendor/lib/libwms.so
-vendor/lib/libwqe.so
-vendor/lib/libxml.so
-vendor/lib/mediadrm/libwvdrmengine.so
 vendor/lib/mmi_audio.so
```

Comment:
Are all mmi_ files are used?

Reply:
I don't know about that, last time I looked in potter vendor they were updated from stock Nougat